tools/protoxform_test_helper.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+from run_command import runCommand
+
+import logging
+import os
+import re
+import sys
+
+
+def PathAndFilename(label):
```

Comment:
Can you add comments to this file? I don't mind if they are simple `#` comments above each function or full Pydoc, as long as we have some way to understand what these helpers do.